<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetExplorer</class>
 <widget class="QDialog" name="SetExplorer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1500</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1500</width>
     <height>800</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="pickASet">
   <property name="geometry">
    <rect>
     <x>82</x>
     <y>74</y>
     <width>281</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black;
font:60px;
qproperty-wordWrap: true;
font-family: fontSpaceGrotesk;</string>
   </property>
   <property name="text">
    <string>Pick a set:</string>
   </property>
  </widget>
  <widget class="QLabel" name="set1">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>160</y>
     <width>307</width>
     <height>164</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="set5">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>348</y>
     <width>307</width>
     <height>164</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="set6">
   <property name="geometry">
    <rect>
     <x>415</x>
     <y>348</y>
     <width>307</width>
     <height>164</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="set7">
   <property name="geometry">
    <rect>
     <x>749</x>
     <y>348</y>
     <width>307</width>
     <height>164</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="set8">
   <property name="geometry">
    <rect>
     <x>1084</x>
     <y>348</y>
     <width>307</width>
     <height>164</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="set9">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>537</y>
     <width>307</width>
     <height>164</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="set10">
   <property name="geometry">
    <rect>
     <x>415</x>
     <y>537</y>
     <width>307</width>
     <height>164</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="set11">
   <property name="geometry">
    <rect>
     <x>749</x>
     <y>537</y>
     <width>307</width>
     <height>164</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="set12">
   <property name="geometry">
    <rect>
     <x>1084</x>
     <y>537</y>
     <width>307</width>
     <height>164</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="set2">
   <property name="geometry">
    <rect>
     <x>415</x>
     <y>160</y>
     <width>307</width>
     <height>164</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="set3">
   <property name="geometry">
    <rect>
     <x>749</x>
     <y>160</y>
     <width>307</width>
     <height>164</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="set4">
   <property name="geometry">
    <rect>
     <x>1084</x>
     <y>160</y>
     <width>307</width>
     <height>164</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
